app/views/api/v2/hosts/base.json.rabl

```diff
@@ -1,3 +1,11 @@
 object @host
 
-attributes :name, :id
+attributes :id
+
+node :name do |name|
+    if Setting[:append_domain_name_for_hosts]
```

Comment:
Nit: Looks like tab is four spaces, instead of two

Reply:
Fixed.